.travis.yml

```diff
@@ -9,13 +9,15 @@ before_cache:
 
 language: python
 python:
-  - "3.7"
+  - "3.9"
 
 before_install:
   - python -m pip install --upgrade virtualenv
 
 install:
   - pip install --upgrade pip
+  - pip install -r requirements.txt
+  - ansible-galaxy collection install community.general
```

Comment:
we're going to drop these usages of `community.general` from this pr right as we are no longer using this module?